deploy/role.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
I don't think this role.yaml file is being used; the Makefile creates deploy/kubedirector/rbac.yaml from deploy/kubedirector/rbac-default.yaml and then uses that rbac.yaml file.

Reply:
The role.yaml file has been deleted.